web_src/js/components/RepoActionView.vue

```diff
@@ -574,14 +670,28 @@ export function ansiLogToHTML(line) {
   flex-direction: column;
 }
 
+.full.height > .action-view-right {
```

Comment:
Does this take affect? It is in `scoped`.

Reply:
This is a good point. It is actually working.. Otherwise, `show fullscreen` will not working.

Reply:
But how could it work? It's out of my knowledge.

IIRC, after Vue compiler's processing, the style selector becomes `.full.height[data-xxx] > ...`  ?

Reply:
Yes I remembered the same thing..

Reply:
<img width="499" alt="Screen Shot 2023-05-30 at 09 02 46" src="https://github.com/go-gitea/gitea/assets/17645053/d0c3e27a-89b2-48e6-8808-99a039c1fc56">

I see, it is because  `full height` is not compiled by Vue, only `action-view-right` is.